draft-ietf-ppm-dap.md

```diff
@@ -827,8 +826,11 @@ The `nonce` and `extensions` fields of each ReportShare match that in the Report
 uploaded by the client. The `encrypted_input_share` field is the `HpkeCiphertext`
 whose index in `Report.encrypted_input_shares` is equal to the index of the aggregator
 in the task's `aggregator_endpoints` to which the AggregateInitializeReq is being sent.
-The `agg_param` field is an opaque, VDAF-specific aggregation parameter. The
-`job_id` parameter contains the leader's chosen AggregationJobID.
+The `agg_param` field is an opaque, VDAF-specific aggregation parameter which is constant per collection flow.
+Note that this is the same value as in `CollectReq` (see {{collect-init}} below) and `AggregateShareReq` (see {{collect-aggregate}} below).
```

Comment:
Can we move this note down to these sections? That is, when describing `CollectReq`, we can place a reference back to this section. Likewise for `AggregateShareReq`.

Reply:
Done.